docs/repo/property-pages/how-to-add-a-launch-profile.md

```diff
@@ -0,0 +1,104 @@
+
+Add the following item to your .props or .targets file:
+
+``` xml
+<PropertyPageSchema Include="path\to\MyLaunchProfile.xaml">
+  <Context>Project</Context>
+</PropertyPageSchema>
+```
+
+### Step 4: Describe the property page
+
+Replace the contents of MYlaunchProfile.xaml with the following:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<Rule Name="MyLaunchProfile"
+      Description="Properties associated with launching and debugging a custom debug target."
+      DisplayName="MyCustomDebugTarget"
+      PageTemplate="commandNameBasedDebugger"
+      xmlns:sys="clr-namespace:System;assembly=mscorlib"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+    <Rule.Metadata>
+        <sys:String x:Key="CommandName">MyCustomDebugTarget</sys:String>
+
+        <!-- KnownImageIds.ImageCatalogGuid -->
+        <sys:Guid x:Key="ImageMonikerGuid">AE27A6B0-E345-4288-96DF-5EAF394EE369</sys:Guid>
+
+        <!-- KnownImageIds.Execute -->
+        <sys:Int32 x:Key="ImageMonikerId">1173</sys:Int32>
+    </Rule.Metadata>
+
+    <Rule.DataSource>
+        <DataSource Persistence="LaunchProfile"
+                    HasConfigurationCondition="False"
+                    ItemType="LaunchProfile"/>
+    </Rule.DataSource>
+
+    <StringProperty Name="ExecutablePath"
+                    DisplayName="Executable"
+                    Description="Path to the executable to run."
+                    Subtype="file" />
+</Rule>
+```
+
+The format of the file is described in detail in [Property Specification](property-specification.md), but the most important points are:
+- The `Name` must be unique.
+- The `PageTemplate` attribute must have the value `"commandNameBasedDebugger"`.
+- The `Rule.Metadata` element must specify a `CommandName`: this is the debug command name (also known as a debug target) corresponding to this UI. Debug commands and how they are handled by VS are beyond the scope of this document,
+- The `ImageMonikerGuid` and `ImageMonikerId` in `Rule.Metadata` together define an ImageMoniker designating the icon to show for this Launch Profile UI. These are optional and if they are not defined a default icon will be supplied.
+- The `Rule.DataSource` element must be specified as shown above.
+  - The `ItemType` attribute indicates that this `Rule` only pertains to launch profiles, as opposed to other types of items that are typically found in a project (`Compile`, `Reference`, etc.).
+  - The `Persistence` attribute indicates that values for the properties should be read and written through the launch profile persistence mechanism, as opposed to trying to read data from the MSBuild file.
+
```

Comment:
Perhaps mention that:
- property categories are ignored for launch profiles, and
- properties are ordered in the UI as they are ordered in this file.